src/rust/en.mangademon/res/source.json

```
@@ -3,8 +3,8 @@
 		"id": "en.mangademon",
 		"lang": "en",
 		"name": "Manga Demon",
-		"version": 2,
-		"url": "https://comicdemons.com",
+		"version": 3,
+		"url": "https://mgdemon.org/",
```

there shouldn't be a trailing slash
```suggestion
		"url": "https://mgdemon.org",
```